sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebViewHelper.java

```diff
@@ -0,0 +1,55 @@
+package com.hcaptcha.sdk;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Color;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+
+import androidx.annotation.NonNull;
+
+import com.hcaptcha.sdk.tasks.OnFailureListener;
+import com.hcaptcha.sdk.tasks.OnLoadedListener;
+import com.hcaptcha.sdk.tasks.OnOpenListener;
+import com.hcaptcha.sdk.tasks.OnSuccessListener;
+
+public final class HCaptchaWebViewHelper {
```

Comment:
`public` can be removed to restrict access to this class from outside. If it makes sense, please do the same for the other classes: `HCaptchaHeadlessWebView`, `HCaptchaDialogFragment`.

Reply:
Done, removed from all internal classes